docs/ssh/main.md

```diff
@@ -70,7 +70,7 @@ After Duo's multi-factor authentication (MFA), you land on one of the many RCC's
 	See [Advanced SSH options](./advance.md) to read more about different arguments you can add to your SSH commands. 
 	
 ### Login nodes	
-Login nodes are the "foyer" of the RCC's clusters. They are connected to the internet and enable you to transfer data to and from the system. They are not designed to carry out computing processes, and you should **NOT run your scripts on login nodes**. To connect to **compute nodes** to run computationally intensive programs, there is one more step you need to go through. 
+Login nodes are the "foyer" of the RCC's clusters. They are connected to the internet and enable you to transfer data to and from the system. They are not designed to carry out computing processes, and you should **NOT run your calculations on login nodes**. To connect to **compute nodes** to run computationally intensive programs, there is one more step you need to go through. 
```

Comment:
- "internet" => "Internet"?
- "computing processes" => "heavy workloads"?
- "NOT run your calculations on login nodes" => "NOT run your heavy workloads on login nodes"?
- "computationally intensive programs" => "computationally intensive work"?

Reply:
corrected